src/portal/src/lib/components/endpoint/endpoint.component.html

```diff
@@ -33,7 +33,7 @@
                         <div *ngSwitchCase="'unknown' || ''"><clr-icon shape="exclamation-triangle" class="is-warning text-alignment" size="22"></clr-icon> Unknown</div>
                     </clr-dg-cell>
                     <clr-dg-cell class="flex-min-width">{{t.url}}</clr-dg-cell>
-                    <clr-dg-cell>{{t.type}}</clr-dg-cell>
+                    <clr-dg-cell>{{getAdapterText(t.type)}}</clr-dg-cell>
```

Comment:
It is recommended to use ’Pipe‘ instead of function, this performance has improved

Reply:
Surely. But for now, it's not widely used. Only for these two components